plugins/repository-azure/src/test/java/org/opensearch/repositories/azure/AzureBlobContainerRetriesTests.java

```diff
@@ -282,7 +282,7 @@ public void testReadRangeBlobWithRetries() throws Exception {
                         exchange.getResponseHeaders().add("Content-Type", "application/octet-stream");
                         exchange.getResponseHeaders().add("Content-Length", String.valueOf(length));
                         exchange.getResponseHeaders()
-                            .add("Content-Range", String.format("bytes %d-%d/%d", rangeStart, rangeEnd.get(), bytes.length));
```

Comment:
Using format + `%d` was not a good idea - the locale is being taken into account